python/cogroo4py/cogroo.py

```diff
@@ -1,402 +1,393 @@
+
+        if type(text) is not Document:
+            doc = self.analyze(text)
+        else:
+            doc = text
+
+        if doc is None:
+            return ''
+
+        ret = []
+        for sentence in doc.sentences:
+            if type_ == 'normal':
+                chunks = sentence.chunks
+            else:
+                chunks = sentence.synchunks
+
+            for chunk in chunks:
+                tokens = []
+                for token in chunk.tokens:
+                    tokens.append(token.lexeme)
+
+                ret.append(chunk.tag + '[' + ' '.join(tokens) + ']')
+
+        return ' '.join(ret)
+
+    pos_tags = {
+        "n": "substantivo",
+        "prop": "nome próprio",
+        "art": "artigo",
+        "pron": "pronome",
+        "pron-pers": "pronome pessoal",
+        "pron-det": "pronome determinativo",
+        "pron-indp": "substantivo/pron-indp",
+        "n-adj": "substantivo/adjetivo",
+        "v": "verbo",
+        "v-fin": "verbo finitivo",
+        "v-inf": "verbo infinitivo",
+        "v-pcp": "verbo particípio",
+        "v-ger": "verbo gerúndio",
+        "num": "numeral",
+        "prp": "preposição",
+        "adj": "adjetivo",
+        "conj": "conjunção",
+        "conj-s": "conjunção subordinativa",
+        "conj-c": "conjunção coordenativa",
+        "intj": "interjeição",
+        "adv": "advérbio",
+        "xxx": "outro"
+    }
+
+    feat_tags = {
+        "M": "masculino",
+        "F": "feminino",
+        "M/F": "masculino/feminino",
+
+        "S": "singular",
+        "P": "plural",
+
+        "NOM": "nominativo",
+        "ACC": "acusativo",
+        "DAT": "dativo",
+        "PIV": "prepositivo",
+
+        "1": "primeira pessoa",
+        "2": "segunda pessoa",
+        "3": "terceira pessoa",
+
+        "1S": "primeira pessoa singular",
+        "2S": "segunda pessoa singular",
+        "3S": "terceira pessoa singular",
+        "1P": "primeira pessoa plural",
+        "2P": "segunda pessoa plural",
+        "3P": "terceira pessoa plural",
+
+        "PR": "presente",
+        "IMPF": "imperfeito",
+        "PS": "perfeito simples",
+        "MQP": "mais-que-perfeito",
+        "FUT": "futuro",
+        "COND": "condicional",
+
+        "IND": "indicativo",
+        "SUBJ": "subjuntivo",
+        "IMP": "imperativo",
+
+        "xxx": "outro"
+    }
+
+    chunk_tags = {
+        "NP": "nominal",
+        "VP": "verbal",
+        "PP": "preposicional",
+        "ADVP": "adverbial",
+        "xxx": "outro"
+    }
+
+    synchunk_tags = {
+        "ACC": "objeto direto",
+        "ADVL": "adjunto adverbial",
+        "APP": "aposição",
+        "DAT": "objeto indireto pronominal",
+        "OC": "predicativo do objeto",
+        "P": "predicado",
+        "PIV": "objeto preposicional",
+        "SA": "complemento adverbial",
+        "SC": "predicativo do sujeito",
+        "SUBJ": "sujeito",
+        "xxx": "outro"
+    }
+
+
+def main():
+    cogroo = Cogroo()
```

Comment:
can we remove this part, or if you think it's helpful to keep it I suggest to add a comment here to clarify why this is here